src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/DatepickerComponent.tsx

```diff
@@ -84,10 +84,16 @@ function DatepickerComponent(props: IDatePickerProps) {
   const [date, setDate] = React.useState<moment.Moment>(null);
   const [validDate, setValidDate] = React.useState<boolean>(true);
   const [validationMessages, setValidationMessages] = React.useState<IComponentBindingValidation>(null);
-  const minDate = props.minDate || DatePickerMinDateDefault;
-  const maxDate = props.maxDate || DatePickerMaxDateDefault;
-  const locale = window.navigator?.language || (window.navigator as any)?.userLanguage || 'nb-NO';
+  const locale = window.navigator?.language || (window.navigator as any)?.userLanguage || 'nb';
   moment.locale(locale);
+
+  const minDate = getFlagBasedDate(props.minDate as DateFlags) ||
+    getISOString(props.minDate) ||
+    DatePickerMinDateDefault;
+  const maxDate = getFlagBasedDate(props.maxDate as DateFlags) ||
+    getISOString(props.minDate) ||
```

Comment:
```suggestion
    getISOString(props.maxDate) ||
```

Reply:
Ah good find, I didnt pick that up when testing. Does your suggestion fix the bug?

Reply:
Yeah, it appears so 👍 Will verify with cypress tests once pushed.